src/GitHub.Exports.Reactive/Services/IGitClient.cs

```diff
@@ -25,20 +25,12 @@ public interface IGitClient
         /// <returns></returns>
         Task Push(IRepository repository, string branchName, string remoteName);
 
-        /// <summary>
-        /// Fetches the remote.
-        /// </summary>
-        /// <param name="repository">The repository to pull</param>
-        /// <param name="remoteName">The name of the remote</param>
-        /// <returns></returns>
-        Task Fetch(IRepository repository, string remoteName);
-
         /// <summary>
         /// Fetches from the remote, using custom refspecs.
         /// </summary>
         /// <param name="repository">The repository to pull</param>
         /// <param name="remoteName">The name of the remote</param>
-        /// <param name="refspecs">The custom refspecs</param>
+        /// <param name="refspecs">The custom refspecs no none to use the default</param>
```

Comment:
> The custom refspecs no none to use the default

I think i know what you wanted to say here, but I feel like you forgot a word or two...

Reply:
That was supposed to be:  "The custom refspecs *or* none to use the default", but I'm not very good with words. Fell free to propose something clearer. 😄

Reply:
I just put the `or` in.